source/_integrations/simplisafe.markdown

```diff
@@ -92,66 +92,6 @@ For any property denoting a volume, the following values should be used:
 
 ## Events
 
-### `SIMPLISAFE_EVENT`
```

Comment:
In the old PR that targeted next branch we removed this text already. Hasn't that PR gone out to `current` yet

Reply:
It doesn't look like `next` has been merged into `current`. The live website still shows the old docs:

![image](https://user-images.githubusercontent.com/47216/117459867-1bbefd00-af09-11eb-9a32-5a9461ff5106.png)

Reply:
Maybe we should have tagged #17643 for the release? The missing tag could be the reason it wasn't cherry picked for the docs rc branch.

@frenck what is the best approach at this point?

Reply:
I have manually cherry-picked #17643 to `current`.

@bachya This PR, of course, not causes a merge conflict. Anything in this PR that still needs to go into current?

Reply:
@frenck Rebased. 👍🏻